manifests/repository.pp

```diff
@@ -50,7 +50,7 @@
     if $subscription {
```

Comment:
Is this subsciption thing something that only exist with bareos < 20?  If no we can maybe simplify these conditionals?

Reply:
Subscription is their premium functionality and does definitely still exist with version 20.

Reply:
I actually found the subscription repository here:
https://download.bareos.com/bareos/release/20/Debian_10/

To compare here is the community repository
https://download.bareos.org/bareos/release/20/Debian_10/

looks like the only difference between the two are .org and .com


Here is the key for the subscription repository I'll update this merge request
```
ryant@cluster01-rya:~ ➜ wget https://download.bareos.com/bareos/release/20/Debian_10/Release.key
--2021-12-15 23:04:17--  https://download.bareos.com/bareos/release/20/Debian_10/Release.key
Resolving download.bareos.com (download.bareos.com)... 185.170.114.121, 2a03:4000:15:27f::1
Connecting to download.bareos.com (download.bareos.com)|185.170.114.121|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1631 (1.6K)
Saving to: 'Release.key’

Release.key                                                                   100%[=================================================================================================================================================================================================>]   1.59K  --.-KB/s    in 0s

2021-12-15 23:04:19 (106 MB/s) - 'Release.key’ saved [1631/1631]

ryant@cluster01-rya:~ ➜ gpg --show-keys Release.key
pub   rsa4096 2020-12-11 [SC]
      C68B001F74D2F20243D0B7A20CCBA537DBE083A6
uid                      Bareos 20 Signing Key <signing@bareos.com>

ryant@cluster01-rya:~ ➜
```